app/src/main/res/values/styles.xml

```diff
@@ -32,4 +32,11 @@
         <item name="android:layout_margin">@dimen/spacing_normal</item>
         <item name="android:backgroundTint">@color/colored_button</item>
     </style>
+
+
```

Comment:
Po co tyle przerwy???? :D

Reply:
To chyba po mergach upstreamów

Reply:
A bywa i tak! Każdy dodał swoją linię i wszystkie są tu odzwierciedlone :P 
Można się ich pozbyć! :)